RELEASE.md

```diff
@@ -0,0 +1,40 @@
+# Release
+Follow the steps outlined in this document to release a new version of `cloudgov-style`.
+
+### Blah blah
+- Releases should follow [semantic versioning](http://semver.org/).
+- The Github tags for the release should just be the number, no `v` before.
+
+## Steps
+
+### Preparation
+1. Decide on a "main idea" for the release, what the release is being done for. Look at previous releases for examples.
+2. Decide on whether the release should be a patch, minor or major version update, based on semver.
```

Comment:
Are we still following semver before we release `1.0.0`? That is to say, as long as the next release has no breaking changes and has additional features it will be `0.5.0`?

Relatedly, will the next breaking change be published as `1.0.0`?